packages/client/src/components/Header.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+function logout() {
+	console.log('logout');
```

Comment:
eslint rules prevent console in CI (we can enable for local builds just be warned)